docs/VnVPlan/VnVPlan.tex

```diff
@@ -379,6 +385,78 @@ \subsection{Traceability Between Test Cases and Requirements}
 \wss{Provide a table that shows which test cases are supporting which
 requirements.}
 
+\setlength{\tabcolsep}{1pt}
+\newgeometry{margin=2cm}
+\begin{landscape}
+\begin{table}[H]
+    \begin{tabular}{|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|}
+      \hline
+      & \textbf{FR1} & \textbf{FR2} & \textbf{FR3} & \textbf{FR4} & \textbf{FR5} & \textbf{FR6} & \textbf{FR7} & \textbf{FR8} & \textbf{FR9} & \textbf{LF-AP1} & \textbf{LF-ST1} & \textbf{UH-EU1} & \textbf{UH-EU2} & \textbf{UH-LR1} & \textbf{UH-LR2} & \textbf{UH-UP1} & \textbf{UH-UP2} \\
+      \hline
+      \textbf{TC-1} & & & & & & & & & & & & & & & & &\\
```

Comment:
I've put TC-1 in all the tables, but its just a placeholder, should be replaced with the real test case later obviously

Reply:
and to be clear I expect an X or some kind of identifier to be put in the columns of the requirements that the given requirement addresses